Gamecube/menu/LoadRomFrame.cpp

```diff
@@ -175,8 +183,16 @@ void Func_LoadFromUSB()
 	// Make sure the romFile system is ready before we browse the filesystem
 	isoFile_init( isoFile_topLevel );
 	
-	pMenuContext->setActiveFrame(MenuContext::FRAME_FILEBROWSER,loadRomMode);
-	fileBrowserFrame_OpenDirectory(isoFile_topLevel);
+	if(Autoboot)
+	{
+		strncpy(isoFile_topLevel->name, AutobootPath, sizeof(isoFile_topLevel->name));
+		fileBrowserFrame_OpenDirectory(isoFile_topLevel);
+	}
+	else
+	{
+		pMenuContext->setActiveFrame(MenuContext::FRAME_FILEBROWSER,loadRomMode);
+		fileBrowserFrame_OpenDirectory(isoFile_topLevel);
```

Comment:
`fileBrowserFrame_OpenDirectory` is called in both branches of the if(), so it can be factorized.

The whole if() can be factorized as well since it's duplicated between `Func_LoadFromSD` and `Func_LoadFromUSB`.